CalibFormats/SiStripObjects/interface/SiStripClusterizerConditionsGPU.h

```diff
@@ -0,0 +1,139 @@
+#ifndef CalibFormats_SiStripObjects_SiStripClusterizerConditionsGPU_h
+#define CalibFormats_SiStripObjects_SiStripClusterizerConditionsGPU_h
+
+#include "DataFormats/SiStripCluster/interface/SiStripTypes.h"
+
+#include "HeterogeneousCore/CUDACore/interface/ESProduct.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/device_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/host_unique_ptr.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/HostAllocator.h"
+
+class SiStripQuality;
+class SiStripGain;
+class SiStripNoises;
+
+namespace stripgpu {
+  static constexpr int kStripsPerChannel = 256;
```

Comment:
couldn't this be `sistrip::APVS_PER_CHAN*sistrip::STRIPS_PER_APV` ? 

https://github.com/cms-sw/cmssw/blob/7f23063d7eff99ab5944dd30b9d88faf64d4ff43/DataFormats/SiStripCommon/interface/ConstantsForHardwareSystems.h#L38-L43

Reply:
@mmusich thanks for the pointer to the ConstantsForHardwareSystems.h header, which I had somehow overlooked.  I have removed the local constants and replaced them with the constants from that header.